Fixed formatting in about and fixed instruction typos per bug 2502. (#2503)
exercism/python: exercises/concept/tisbury-treasure-hunt/.docs/instructions.md

# Instructions

Aazra and Rui are teammates competing in a pirate-themed treasure hunt.

>   One has a list of treasures with map coordinates, the other a list of location names with map coordinates.
>   They've also been given blank maps with a starting place marked YOU ARE HERE.

Reviewer: ```suggestion
One has a list of treasures with map coordinates, the other a list of location names with map coordinates.
They've also been given blank maps with a starting place marked YOU ARE HERE.
```

<br>

But things are a bit disorganized: Azara's coordinates appear to be formatted and sorted differently from Rui's, and they have to keep looking from one list to the other to figure out which treasures go with which locations.

>   Being budding pythonistas, they've come to you for help in writing a small program (a set of functions, really) to better organize their hunt information.

Reviewer: ```suggestion
  Being budding pythonistas, they have come to you for help in writing a small program (a set of functions, really) to better organize their hunt information.
```

>   Being budding pythonistas, they have come to you for help in writing a small program (a set of functions, really) to better organize their hunt information.

Reviewer: ```suggestion
Being budding pythonistas, they have come to you for help in writing a small program (a set of functions, really) to better organize their hunt information.
```

## 4. Combine matched records

> Implement the `create_record()` function that takes a `(treasure, coordinate)` pair from Azaras list and a `(location, coordinate, quadrant)` record from Ruis list and returns `(treasure, coordinate, location, coordinate, quadrant)` **if the coordinates match**. If the coordinates _do not_ match, return the string **"not a match"**

Reviewer: ```suggestion
Implement the `create_record()` function that takes a `(treasure, coordinate)` pair from Azaras list and a `(location, coordinate, quadrant)` record from Ruis' list and returns `(treasure, coordinate, location, coordinate, quadrant)` **if the coordinates match**. If the coordinates _do not_ match, return the string **"not a match"**
```

>   Re-format the coordinate as needed for accurate comparison.

Reviewer: ```suggestion
Re-format the coordinate as needed for accurate comparison.
```

```python
>>> create_record(('Brass Spyglass', '4B'),('Abandoned Lighthouse', ("4", "B"), 'Blue'))
('Brass Spyglass', '4B', 'Abandoned Lighthouse', ("4", "B"), 'Blue')

>>> create_record(('Brass Spyglass', '4B'),(("1", "C"), 'Seaside Cottages', 'blue'))
"not a match"
```

## 5. "Clean up" & make a report of all records

Clean up the combined records from Azara and Rui so that there's only one set of coordinates per record. Make a report so they can see one list of everything they need to put on their maps.

>   Implement the `clean_up()` function that takes a tuple of tuples (_everything from both lists_), looping through the _outer_ tuple, dropping the unwanted coordinates from each _inner_ tuple and adding each to a 'report'.
>   Format and return the 'report' so that there is one cleaned record on each line.

Reviewer: ```suggestion
Implement the `clean_up()` function that takes a tuple of tuples (_everything from both lists_), looping through the _outer_ tuple, dropping the unwanted coordinates from each _inner_ tuple and adding each to a 'report'.
Format and return the 'report' so that there is one cleaned record on each line.
```

If these have two spaces before the sentences because they are meant to be paragraphs, then the effect would be accomplished by an additional new line instead of the two leading white spaces.
